date_range/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Date Range",
     "summary": "Manage all kind of date range",
-    "version": "14.0.2.1.3",
+    "version": "14.0.2.1.4",
```

Comment:
@abenzbiria The version bump will be done at merge time. Can you remove this change PLZ?